app/services/scrape_result_search_service.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class ScrapeResultSearchService
```

Comment:
That should not be a service but instead a query object.

Service objects should be reserved for all complex operations except read if the data comes from directly a database.

Reply:
👍 . Fixed in 0c51983d0128e1c889b63a9fcbe35790f2554c77